FreeRTOS/FreeRTOS-Kernel
# Removing issue template for general inquiries.

Description
-----------
Removing issue template for general inquiries. Users with general inquiries are redirected to forum.

Test Steps
-----------
Please see https://github.com/FreeRTOS/FreeRTOS/pull/105

Related Issue
-----------
N/A


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
